packages/xo-web/src/xo-app/logs/backup-ng/log-alert-body.js

```diff
@@ -188,7 +189,8 @@ TaskInfos.propTypes = {
 
 const VmTask = ({ children, className, restartVmJob, task }) => (
   <li className={className}>
-    <Vm id={task.data.id} name={task.data.name_label} link newTab /> <TaskStateInfos status={task.status} />{' '}
+    <Vm id={task.data.id} name={task.data.name_label ?? VM_NOT_PRESENT} link newTab />{' '}
```

Comment:
I don't understand in which case data.name_label is undefined. Is it set during backup or is it dynamically added by the front ?

Reply:
It is dynamically added by the front. I am searching for a way to add the `VM_NOT_PRESENT` value where `data.name_label` is set, but I am not very familiar with this piece of code, so I'm struggling a bit

Reply:
I feel like if we want to add a custom message for this situation, and avoid this message to show up in other places, what I did is the best way to do it.